pkg/internal/export/otel/traces.go

```diff
@@ -58,6 +58,10 @@ type TracesConfig struct {
 
 	// Grafana configuration needs to be explicitly set up before building the graph
 	Grafana *GrafanaOTLP `yaml:"-"`
+
+	// Include SQL statement in traces
+	IncludeDBStatement       bool `yaml:"include_db_statement" env:"BEYLA_INCLUDE_DB_STATEMENT"`
```

Comment:
I created our own option, because GO_AUTO will not make sense when we add SQL support for kprobes.